### ISSUE TITLE
mark_tutorial_complete..?

### ISSUE THREAD

**Issue by nicarmani** [maierfelix/POGOserver#192]
Ok after a while i fully complete the setup but cant join into the game..?

POGOserver 0.5.9

![06769d9917af825544f2248046f580ac](https://cloud.githubusercontent.com/assets/19747271/18601422/1bc33f3a-7c39-11e6-978f-7bfd55432a89.png)

![6e1b1c088ca9f09e3e02f630673ae7e9](https://cloud.githubusercontent.com/assets/19747271/18601438/3165e7fc-7c39-11e6-8652-aea2910ceb7a.png)

**Comment by nogard230** [comment]
Hi, Update mysql:
UPDATE `users` SET `tutorial` = '{"0":1,"1":1,"3":1,"4":1,"7":1}' WHERE `users`.`id` = *;

and try again.

**Comment by Cronick** [comment]
Try to delete all of your tables, and restart the server.

**Comment by nicarmani** [comment]
yup, that works partially.. delete the database works well once, but when I restart the console happen again after a crash. may I have something wrong in my setup?
